test.js

```diff
@@ -34,11 +34,28 @@ function jasmine(options) {
 }
 
 test('run unit test and pass', async t => {
-	const stdout = await jasmine({timeout: 9000, verbose: true});
+	const stdout = await jasmine('fixture.js', {timeout: 9000, verbose: true});
 
 	t.true(/should pass: passed/.test(stdout));
 });
 
+test('run unit test and fail silently', async t => {
+	const stdout = await jasmine('fail-fixture.js', {timeout: 9000, verbose: true, noFail: true});
+
+	t.true(/should fail: failed/.test(stdout));
+});
+
+test('run unit test and fail', async t => {
+	let exceptionThrown = 0;
+	try {
+		await jasmine('fail-fixture.js', {timeout: 9000, verbose: true});
+	} catch (ex) {
+		exceptionThrown++;
+	}
+
+	t.true(exceptionThrown === 1);
```

Comment:
Ok, never used this framework before. I understand why the tests were switched to this framework, but I must say that mocha made more sense to me as a testing framework since the syntax was at least similar to jasmine. This seems like an unnecessary hindrance for would-be contributors. Just my 2c.